app/views/activities/show.html.slim

```diff
@@ -1,47 +1,73 @@
 .h1.pb-0 = @activity.label
+- @activity.tags.each do |tag|
+  i class="#{tag.icon}"
 
 .card.mt-3.mb-3
   .card-body
-    dl
-      .row
-        .col-12
-          dt = t('activerecord.attributes.activity.location')
-          dd = @activity.location
-      .row
-        .col-12
-          dt = t('activerecord.attributes.activity.description')
-          dd = @activity.description
-
-      .row
-        .col-6
-          dt = t('activerecord.attributes.activity.language')
-          dd = t("simple_form.options.activity.language.#{@activity.language}")
-        .col-6
-          dt = t('activerecord.attributes.activity.js_type')
-          dd = t("simple_form.options.activity.js_type.#{@activity.js_type}")
-
-      .row
-        .col-6
-          dt = t('activerecord.attributes.activity.participants_count_activity')
-          dd = @activity.participants_count_activity
-        .col-6
-          dt = t('activerecord.attributes.activity.participants_count_transport')
-          dd = @activity.participants_count_transport
-
-      .row
-        .col-6
-          dt = t('activerecord.attributes.activity.duration_activity')
-          dd = @activity.duration_activity
-        .col-6
-          dt = t('activerecord.attributes.activity.duration_journey')
-          dd = @activity.duration_journey
-
-      .row
-        .col-12
-          dt = t('activerecord.attributes.activity.activity_documents')
+   = @activity.description
+
+.card.mt-3.mb-3
+  .card-body
+    .row
+      .col-6
+        dt = t('activerecord.attributes.activity.language')
+        dd = t("simple_form.options.activity.language.#{@activity.language}")
+
+        dt = t('activerecord.attributes.activity.location')
+        dd = @activity.location
+
+      .col-6
+        dt = t('activerecord.attributes.activity.goals')
+        dd
+          ul
+            - @activity.goals.each do |s|
+              li = s.name
+.card.mt-3.mb-3
+  .card-body
+    .row
+      .col-6
+        dt = t('activerecord.attributes.activity.js_type')
+        dd = t("simple_form.options.activity.js_type.#{@activity.js_type}")
```

Comment:
right now it can be also `nil` which is ok i guess? but then this part does not work:
```suggestion
        dd = t("simple_form.options.activity.js_type.#{@activity.js_type || 'none'}")
```

or what i usually do:

```suggestion
        dd = t(@activity.js_type || 'none', scope: 'simple_form.options.activity.js_type')
```


```
simple_form.options.activity.js_type.none = 'Keiner'
```
        
        
```